pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -146,6 +147,13 @@ type ConfigFile struct {
 		// Specifying ApiEndpoint will override the default GCE compute API endpoint.
 		ApiEndpoint string `gcfg:"api-endpoint"`
 		LocalZone   string `gcfg:"local-zone"`
+		// location is the KMS location of the KeyRing to be used for encryption. The default value is "global". // It can be found by checking the available KeyRings in the IAM UI.
+		// This is not the same as the GCP location of the project.
+		Location string `gcfg:"kms-location,omitempty"`
+		// keyRing is the keyRing of the hosted key to be used. The default value is "google-kubernetes".
+		KeyRing string `gcfg:"kms-keyRing"`
+		// cryptoKey is the name of the key to be used for encryption of Data-Encryption-Keys.
+		CryptoKey string `gcfg:"kms-cryptoKey"`
```

Comment:
nit: nothing else in this struct uses mixed case... is this reading an existing format or making up its own?